saleor/graphql/core/fields.py

```diff
@@ -186,9 +187,13 @@ def connection_resolver(
         filter_input = args.get(filters_name)
 
         if filter_input and filterset_class:
-            iterable = filterset_class(
+            instance = filterset_class(
                 data=dict(filter_input), queryset=iterable, request=info.context
-            ).qs
+            )
+            # Make sure filter input has valid values
+            if not instance.is_valid():
+                raise GraphQLError("You must provide valid fields.")
```

Comment:
I think this is fine, but was wondering if we can use `instance.errors` to show what was wrong with the values. The thing is it's an error dict and there may be multiple errors.

Reply:
Please check again. Using like this `GraphQLError(instance.errors.as_data())`,  we got a message error like this `{'ids': [ValidationError(['Invalid ID specified.'])]}`. :)

Reply:
I think it's fine now. In the future, I would think about improving it to return messages formatted as "field: message", but it's good enough already.

Reply:
I'm going to check if we already have a method to format those messages and if not what we can do. ;)

Reply:
Using `get_json_data` we got a result like this `{'ids': [{'message': 'Invalid ID specified.', 'code': ''}]}`. Please let me know if it better. ;)